src/main/java/micdoodle8/mods/galacticraft/api/spaceprojects/SpaceProject.java

```diff
@@ -0,0 +1,46 @@
+package micdoodle8.mods.galacticraft.api.spaceprojects;
+
+import java.util.HashMap;
+import java.util.UUID;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.server.MinecraftServer;
+
+public class SpaceProject {
+    // Maps UUIDs to other UUIDs. This allows users to join a team.
+    public static HashMap<UUID, UUID> GlobalSpaceProjectTeam = new HashMap<>(100, 0.9f);
```

Comment:
Since you have access methods, shouldn't this be private?

Reply:
No I cannot as its used in GCGlobalVariableWorldSavedData.java directly. In theory i could make an access method, but i don't see a point to that.